coordinator/src/services/translation/cancellation/cancellation-response.ts

```diff
@@ -110,7 +110,7 @@ function createBundleEntries(cancellationResponse: CancellationResponse) {
     const performerAgentPerson = cancellationResponse.performer.AgentPerson
     let performerId
     if (isDeepStrictEqual(performerAgentPerson, hl7ResponsiblePartyAgentPerson)) {
```

Comment:
I think this should be checking hl7AuthorAgentPerson on line 112 and hl7ResponsiblePartyAgentPerson on line 114